ui/app/com/gu/recipeasy/views/admin/index.scala.html

```diff
@@ -1,9 +1,10 @@
-@()(implicit messages: play.api.i18n.Messages, request: RequestHeader)
+@(loggedInUserEmailAddress: String)(implicit messages: play.api.i18n.Messages, request: RequestHeader)
 @layout("Recipeasy Admin") {
 
     <h2>Recipeasy Admin Area</h2>
     <ul>
         <li><a href="@routes.Application.recentActivity">Recent activity</li>
+        <li><a href="@routes.Application.recentUserActivity(loggedInUserEmailAddress)">Recent activity of a specific user</li>
```

Comment:
Would this not be better provided as a form? Allowing the user to provide the email address would remove the choice of "what's the email address we should show first", as well as simplifying the usage.